wpilibc/shared/src/Commands/WaitCommand.cpp

```diff
@@ -9,21 +9,11 @@
 #include <sstream>
 
 WaitCommand::WaitCommand(double timeout)
-    : Command(
+    : TimedCommand(
           ((std::stringstream&)(std::stringstream("Wait(") << timeout << ")"))
```

Comment:
Running that line locally returned `5)it(`. `("Wait(" + std::to_string(timeout) + ")").c_str()` produces `Wait(5.000000)`, which is probably the output you intended.